packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Skip To Content'
+section: 'components'
+cssPrefix: 'pf-c-skip-to-content'
+---
+
+## Simple SkipToContent
```

Comment:
can we make this sentence case.  `skip to content`

Reply:
Done.